src/routes/articles/assignTagToArticle.e2e.test.js

```diff
@@ -0,0 +1,140 @@
+const {expect} = require('chai')
+const {fixtures} = require('../../../test/utils')
+
+
+describe('POST /articles/:id/tags', () => {
+
+	let user
+	let accessToken
+
+	before(async ()=>{
+
+		 user = await fixtures.insertUser() 
+		const body = { id: user.id, email: user.email }
```

Comment:
Hi CTO, I am unable to follow this. is there a reason why it is referred to `body`?

Reply:
Personal choice actually. I thought of `user` being an object and `id`, `email` being fields/body of the `user` object

Reply:
I think we should perhaps use intention revealing names. 
i can't think of a good one other than this name as an example 
eg. `accessTokenParams` 
if you do have one better than this i think we can use.
Any one other than body. just so we know why we are accessing those information from the user.

Reply:
yes please, thanks